src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1753,6 +1755,11 @@ private async Task HandleRpcAsync(JsonRpcMessage rpc)
                         return;
                     }
 
+                    if (request.IsNotification && request.Method == ProgressRequestSpecialMethod)
+                    {
+                        return;
+                    }
+
                     JsonRpcMessage result = await this.DispatchIncomingRequestAsync(request).ConfigureAwait(false);
```

Comment:
I don't think we should swallow this. Folks could be using a formatter that doesn't support this, and they may want to handle it themselves in that case.